### PR TITLE
Refactor migration ibmdb

### PR DESCRIPTION
**Pull request by b-admike**

### CODE REVIEW

File: lib/ibmdb.js [hunk 1]
```diff
@@ -94,6 +94,74 @@ function IBMDB(name, settings) {
...
+
+    models = models || Object.keys(this._models);
+
+    // var changes = [];
+    async.each(models, function(model, done) {
+      self.getTableStatus(model, function(err, fields, indexes) {
+        if (err) {
+          return done(Error(err));
+        }
+        // TODO: VALIDATE fields/indexes against model definition
```
Comment (jannyHou) [inline lib/ibmdb.js]: We can implement empty functions here if details are not provided.
So we can remove `isActual` here, use the one in base connector, and add two empty functions `getAddModifyColumns` and `getColumnsToDrop`

**Comment by ssh24** [comment]
@slnode test please

**Comment by ssh24** [comment]
@slnode test please

**Comment by jannyHou** [comment]
@ssh24 I think we need to create a new file `migration.js` in `loopback-ibmdb` then move all functions there. Instead of defining everything in `ibmdb.js`

**Pull request merged**